### DISCUSSION TITLE
Difference dynamic color for same artwork?

### DISCUSSION THREAD

**Discussion by NikunjK8** [fast4x/RiMusic#1338] [Q&A]
Both songs have same artwork. Only difference is right one have some white text in corner. But dynamic color is different for both. Is it normal?

![IMG_20240401_213904](https://github.com/fast4x/RiMusic/assets/99175083/8fd4efb4-0e5d-40c2-a44c-abe973c99a7d)

**Comment by fast4x** [comment]
Yes it's normal
